packages/core/src/connectors/index.test.ts

```diff
@@ -34,6 +34,12 @@ const aliyunSmsConnector = {
   config: {},
   createdAt: 1_646_382_233_666,
 };
+const appleConnector = {
+  id: 'apple-universal',
+  enabled: false,
+  config: {},
+  createdAt: 1_646_382_233_666,
+};
```

Comment:
@wangsijie @darcyYe like mentioned offline, maybe we could clean up these mocks to the __mock__ files as part of the refactor job.